src/net/fe/FEMultiplayer.java

```diff
@@ -87,7 +88,7 @@ public static void main(String[] args) {
 			game.init(480, 320, "Fire Emblem Multiplayer");
 			/* Testing code */
 //			game.testFightStage();
-//			game.testOverworldStage();
+			game.testOverworldStage();
```

Comment:
Short term: comment this back out.

Long term, this project probably needs more main methods.